libobs/obs.h

```diff
@@ -1996,6 +1996,18 @@ static inline void obs_source_frame_destroy(struct obs_source_frame *frame)
 EXPORT void obs_source_frame_copy(struct obs_source_frame *dst,
 		const struct obs_source_frame *src);
 
+/* ------------------------------------------------------------------------- */
+/* Media controls */
+EXPORT void obs_media_play_pause(obs_source_t *source);
+EXPORT void obs_media_restart(obs_source_t *source);
+EXPORT void obs_media_stop(obs_source_t *source);
+EXPORT void obs_media_next(obs_source_t *source);
+EXPORT void obs_media_previous(obs_source_t *source);
+EXPORT uint32_t obs_media_get_duration(obs_source_t *source);
+EXPORT uint32_t obs_media_get_time(obs_source_t *source);
+EXPORT void obs_media_set_time(obs_source_t *source, uint32_t ms);
+EXPORT enum obs_media_state obs_media_get_state(obs_source_t *source);
+
```

Comment:
These functions should really be in the source section, a sub-section for media functions (if you look in the file, you'll see sub-sections like "functions used by sources").

Additionally, these functions should be renamed.  For example, instead of `obs_media_restart`, I'd probably recommend `obs_source_media_restart`, then similarly for the other functions.